products/cpdb/recipe.yml

```diff
@@ -23,7 +23,8 @@ inputs:
     - name: doitt_buildingfootprints
 
 # Projects
-    - name: cpdb_capital_spending
+    - name: nycoc_checkbook
+      version: 2010-2023
```

Comment:
pinned for now - we should perhaps think a bit more about how we actually want to version this, when we trigger the job, etc